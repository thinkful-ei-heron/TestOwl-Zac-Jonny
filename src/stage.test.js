import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import Stage from './stage';

it('renders without crashing', () => {
	const div = document.createElement('div');
	ReactDOM.render(<Stage />, div);
	ReactDOM.unmountComponentAtNode(div);
});

it('renders the List as expected', () => {
	const stage = renderer
		.create(<Stage />)
		.toJSON();
	expect(stage).toMatchSnapshot();
});