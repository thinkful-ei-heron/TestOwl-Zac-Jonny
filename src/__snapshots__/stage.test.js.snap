// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the List as expected 1`] = `
<section
  className="stage"
>
  <div
    className="tile"
  >
    <h2>
      Koren Templeton
    </h2>
    <img
      alt="The avatar of Koren Templeton"
      src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
    />
  </div>
  <div
    className="tile"
  >
    <h2>
      Ashla Attwool
    </h2>
    <img
      alt="The avatar of Ashla Attwool"
      src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
    />
  </div>
</section>
`;
